Chocolatey.Cake.Recipe/Content/parameters.cake

```diff
@@ -27,17 +27,7 @@ public static class BuildParameters
 {
     private static Func<BuildVersion, object[]> _defaultNotificationArguments = (x) =>
     {
-        var firstPortionOfQueryString = string.Empty;
-
-        if (x.PackageVersion.Contains("-"))
-        {
-            var betaVersionNumber = x.PackageVersion.Substring(x.PackageVersion.IndexOf('-') + 1);
-            firstPortionOfQueryString = betaVersionNumber + "-";
-        }
-
-        var dateForQueryString = DateTime.Now.ToString("MMMMM-d-yyyy").ToLower();
-
-        var arguments = new object[] { x.PackageVersion, string.Format("{0}{1}", firstPortionOfQueryString, dateForQueryString) };
+        var arguments = new object[] { x.PackageVersion, string.Format("v{0}", x.PackageVersion) };
```

Comment:
It could be useful to have the old date part still available as an argument.

Maybe keep it as the third argument (which is unused), especially since at the moment this Func is not overridable.

Reply:
I think that would just complicate things.  We haven't decided to move forward with the version number only, so let's stick with that.  Including the date in this way was also limited in the sense that it _had_ to be run on the day the release was done, which isn't always that case.  If we actually want the date, we should look to something like the date the tag was applied, which would be done under a different issue.